gluten-iceberg/src/test/scala/org/apache/gluten/execution/IcebergSuite.scala

```diff
@@ -21,9 +21,10 @@ import org.apache.gluten.GlutenConfig
 import org.apache.spark.SparkConf
 import org.apache.spark.sql.Row
 
-class VeloxIcebergSuite extends WholeStageTransformerSuite {
-
+abstract class IcebergSuite extends WholeStageTransformerSuite {
   protected val rootPath: String = getClass.getResource("/").getPath
+  // FIXME: This folder is in module backends-velox so is not accessible if profile backends-velox
+  //  is not enabled during Maven build.
   override protected val resourcePath: String = "/tpch-data-parquet"
   override protected val fileFormat: String = "parquet"
```

Comment:
@baibaichen 

We can move `/opt/gluten/backends-velox/src/test/resources/tpch-data-parquet` to module `gluten-substrait` in your PRs if you want to share this test data with Velox backend

Reply:
now ch backend use the `gluten-core/src/test/resources/tpch-data` as the tpch data.

Reply:
> now ch backend use the `gluten-core/src/test/resources/tpch-data` as the tpch data.

Noted. I could open another PR to test iceberg with the data. Not sure why different dataset is used.